atest/testdata/keywords/type_conversion/custom_converters.robot

```diff
@@ -69,6 +69,9 @@ Failing conversion
     Conversion should fail    Strict    wrong type
     ...    type=Strict    error=TypeError: Only Strict instances are accepted, got string.
 
+Only vararg
+    Only var arg    10    ${10}
```

Comment:
This test needs a counterpart under `atest/robot`. See `atest/README.rst` for details.

Reply:
Thanks, done!